calculator.rb

```diff
@@ -0,0 +1,142 @@
+def run_calculator
```

Comment:
I like that you're really comfortable using methods, though strictly speaking this may be a little overboard. You could have left this at the bottom of the file and it would have been just as clear.